app/views/email_templates/update.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>EmailTemplates#update</h1>
+<p>Find me in app/views/email_templates/update.html.erb</p>
```

Comment:
Not entirely sure what this is, can someone explain?

Reply:
It's an auto-generated view by Rails. This should have been deleted. Thanks for catching it!